datafusion/src/datasource/object_store/mod.rs

```diff
@@ -70,7 +70,7 @@ pub enum ListEntry {
 }
 
 /// The path and size of the file.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, Clone, PartialEq, Eq)]
```

Reply:
Not sure if it is really useful to add a trait that is not used 😊